src/tilesource.js

```diff
@@ -806,32 +806,10 @@ $.TileSource.prototype = {
      * @param {string} [context.errorMsg] Private parameter. The final error message, default null (set by finish).
      */
     downloadTileStart: function (context) {
-        const dataStore = context.userData,
```

Comment:
This is the main change that triggered the rest. The idea is: why we force some format on the tile data here? Let's leave this up to the conversion pipeline to decide what is best for the drawer. This might not be obvious now, but for example WebGL can also work with blobs and avoid any conversion middle steps. We can add blob support to webgl later.

Moving more load to the conversion has also shown some issues with the async interaction, mainly in tests (e.g. the viewer gets destroyed and some routines are not yet finished).